templates/init_webservice.sh.template

```diff
@@ -2,6 +2,6 @@
 
 cd "$(dirname "$0")"
 
-java -XX:MaxRAMPercentage=50.0 -XX:+ExitOnOutOfMemoryError -jar dockstore-webservice-*.jar server web.yml | tee --append /dockstore_logs/webservice.out
+java -XX:MaxRAMPercentage=50.0 -XX:+ExitOnOutOfMemoryError -jar /home/dockstore-webservice-*.jar server web.yml | tee --append /dockstore_logs/webservice.out
```

Comment:
I think the 50% is a holdover from when we had ElasticSearch and Postgres running on the same machine. We  probably should have raised this for EC2 (and could maybe have brought down the instance size in conjunction with that).

But not to digress from the Fargate PR, that could be tackled separately... Should we be setting this at all for Fargate? I don't know, just asking.

Reply:
Hmmm, also Nextflow but yes. I suspect we can tune this for Fargate with some performance testing. 
It probably should be a spin-off ticket to be tackled, much later.

Reply:
Ah, right forgot about Nextflow... we probably don't want to go too much higher after all, at least not in the EC2 world.